src/main/java/com/google/sps/model/Trip.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.model;
+
+import java.util.List;
+
+public class Trip {
+    private long tripID;
+    private String title;
+    private List<Long> participants;
+    public float totalCost;
+    private float totalBudget;
+    
+    public long getParticipantAmmountOwed(long userId){
+        //TODO: calculate participant ammount owed
```

Comment:
Where do you store the per participant cost needed for this method?

No need to resolve in this PR.

Reply:
I don't know of a good place to store the per participant cost. I was thinking that when needed it could be calculated by going through each item that had the user in the participants list.